salt/runner.py

```diff
@@ -236,17 +236,17 @@ def run(self):
                     low['kwarg']['orchestration_jid'] = async_pub['jid']
 
                 # Run the runner!
-                if self.opts.get('async', False):
```

Comment:
I think if we leave this as `async` and not switch it over to asyncronous, we shouldn't see anyone affected, because they should be using `self.cmd` and passing `async: True` to self.opts. which is what salt-run --async would do, or using `cmd_async`, which is how all the other python apis work.